wire-ios/THIRD-PARTY-NOTICES.txt

```diff
@@ -161,35 +161,6 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-8) FBSnapshotTestCase under BSD license
```

Comment:
Is this auto generated or manual?

Reply:
Probably it was generated at some point, but for now we keep the update as it is.